docs/cellxgene_census_FAQ.md

```diff
@@ -0,0 +1,109 @@
+# Frequently Asked Questions for CZ CELLxGENE Discover Census
+
+Last updated: Apr, 2023.
+
+- [Why should I use the Census?](#Why-should-I-use-the-Census)
+- [Why does the Census not have a normalized layer or embeddings?](#Why-does-the-Census-not-have-a-normalized-layer-or-embeddings)
+- [How does the Census differentiate from other services?](#How-does-the-Census-differentiate-from-other-services?)
```

Comment:
The link does not work. This should be #how-does-the-census-differentiate-from-other-services. The "?" should not be present.